terraform-ci/pipeline.yaml

```diff
@@ -0,0 +1,39 @@
+steps:
+  - label: ":terraform: :mag: validate"
+    plugins:
+      - docker#v5.9.0:
+          image: hashicorp/terraform:1.5
+          command: ["validate"]
+          environment:
```

Comment:
We can clean this up a bit with the following flag:

https://github.com/buildkite-plugins/docker-buildkite-plugin?tab=readme-ov-file#propagate-aws-auth-tokens-optional-boolean